mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.kt

```diff
@@ -410,7 +409,11 @@ abstract class ContentController protected constructor(private val activity: Mai
      * @return true if back key can be consumed, false otherwise
      */
     fun canGoBack(): Boolean {
-        return temporaryPage != null || !pageStack.empty()
+        return if ((temporaryPage as? WebViewFragment)?.isStackRoot == true) {
+            return (temporaryPage as? WebViewFragment)?.goBack() == true
```

Comment:
`goBack` -> `canGoBack`

Also probably should put the conversion result into an intermediate variable ... doing so is a bit more readable (IMHO) and saves one null check.